### PR TITLE
Fix node version and permission on PrestaShop DockerFile

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix node version and permission on PrestaShop DockerFile
| Type?             | bug fix
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | #25925 
| How to test?      | See #25925 
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25938)

### CODE REVIEW

**Review by matks — APPROVED** [review]

File: .docker/Dockerfile [hunk 1]
```diff
@@ -7,10 +7,12 @@ RUN groupmod -g 1000 www-data \
 COPY /wait-for-it.sh /tmp/
 COPY /docker_run_git.sh /tmp/
 
+RUN mkdir -p /var/www/.composer
 RUN mkdir -p /var/www/.npm
-RUN chown -R www-data:www-data /var/www/.npm
 
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN chown -R www-data:www-data /var/www/
```
Comment (PierreRambaud) [inline .docker/Dockerfile:13]: Not sure we want to change the parent directory as we only want to change .composer and .npm paths, Seems I made the same PR as you #25940 :sweat_smile:

Comment (PierreRambaud) [reply .docker/Dockerfile:13]: Also because the parent directory can be mounted and the whole content is independent from the Docker fs

Comment (boubkerbribri) [reply .docker/Dockerfile:13]: I think there was a problem with building assets, couldn't write files in some directories. That's why.

**Comment by boubkerbribri** [comment]
Closing the PR since it's a duplicate

**Pull request closed without merging**